subgraph/src/public-lock.ts

```diff
@@ -239,7 +238,11 @@ export function handleKeyExtended(event: KeyExtendedEvent): void {
     key.save()
   }
   // create receipt
-  createReceipt(event)
+  if (key && key.owner) {
+    createReceipt(event, key.owner, event.params.tokenId)
+  } else {
+    createReceipt(event, Address.fromString(''), event.params.tokenId)
```

Comment:
not sure when you end up with an empty key owner on renewal? Also this `Address.fromString('')` gives a null address?